controllers/Redirects.php

```diff
@@ -138,6 +138,30 @@ public function update($recordId = null, $context = null)
         parent::update($recordId, $context);
     }
 
+    /**
+     * Force rule flushing on update
+     */
+    public function update_onSave(?string $context = null)
+    {
+        /** @noinspection PhpPossiblePolymorphicInvocationInspection */
```

Comment:
@jaxwilko what is this monstrosity?

Reply:
Basically when a record is updated, if the cache for that record is not cleared then the old redirect lives in cache and wont be updated, it creates a problem for users. This change forces the cache for the specific rule being updated to be flushed, rather than invalidating the entire redirects cache.

It's significantly less than ideal, so very open to other suggestions :)

Reply:
Sorry @jaxwilko I meant the comment, not the overall method. :P